packages/components/src/components/notification/_inline-notification.scss

```diff
@@ -177,6 +177,12 @@
       color: $inverse-link;
     }
 
+    &:focus {
+      border-color: transparent;
+      outline: 2px solid $inverse-focus-ui;
+      outline-offset: -2px;
```

Comment:
With this I made a judgement call, so let me know what you think! I couldn't find an instance of an inverse ghost button, and in this case the border + outline colors were strange when one or the other was `$inverse-focus-ui`. So this makes the focus outline of the button one color, similar to the close buttons.